### DISCUSSION TITLE
Removing window titlebar from applications

### DISCUSSION THREAD

**Discussion by brixel-1984** [lxqt/lxqt#2480] [Q&A] [answered]
Hi,

In Gnome,  the shell extension Unite (https://extensions.gnome.org/extension/1287/unite/) allows you to remove the window titlebar from an application. Is there an equivalent for the LXQT desktop environment.

Thanks

**Comment by tsujan** [comment]
That isn't related to LXQt but to the *window manager* you use with LXQt. KWin has such a setting; some other WMs may do too.

**Reply by stefonarch** [reply, marked as answer]
Openbox in `rc.xml` 

```
<applications>
    <!-- Match all windows and remove their decorations -->
    <application class="*"> <decor>no</decor> </application>
    <!-- Give back decorations to some app windows -->
    <application name="qps"> <decor>yes</decor> </application>
    <application name="featherpad"> <decor>yes</decor> </application>
</applications>
```

It's possible to do [something](https://www.reddit.com/r/xfce/comments/fa9lj3/is_there_a_way_to_completely_hide_the_title_for/) similar in xfwm4 themes too.

**Reply by brixel-1984** [reply]
Thanks!
